lib/utils/paths.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const path = require('path');
+const { BASE_ASSETS, ROOT } = require('../utils/globals');
+
+const pkgLog = ({ org = '', name = '', version = '' } = {}) => {
+    return path.join(ROOT, org, BASE_ASSETS, name, `${version}.log.json`);
+}
+module.exports.pkgLog = pkgLog;
```

Comment:
I'm a bit confused by the names of the utils `paths` and `pathnames`? Which util should be used for what?